apps/emqx_authn/src/simple_authn/emqx_authn_mnesia.erl

```diff
@@ -166,8 +169,9 @@ authenticate(
         password_hash_algorithm := Algorithm
     }
 ) ->
+    Tenant = maps:get(tenant, Credential, undefined),
     UserID = get_user_identity(Credential, Type),
-    case mnesia:dirty_read(?TAB, {UserGroup, UserID}) of
```

Comment:
What if we make use of the `UserGroup` for tenant?
and rename `is_superuser` to `properties` map() field, `is_superuser` being one of the properties, `group` being another.

Reply:
then we do not need to change the table schema, on step close to upgrade from a regular 5.0 database.